test/powershell/Modules/Microsoft.PowerShell.Utility/WebCmdlets.Tests.ps1

```diff
@@ -2332,6 +2332,24 @@ Describe "Invoke-RestMethod tests" -Tags "Feature" {
         }
     }
 
+    Context "Invoke-RestMethod Single Value JSON null support" {
+        BeforeAll {
+            $baseUrl = 'http://localhost:8081/PowerShell?test=response&contenttype=application/json&output='
+        }
+        It "Invoke-RestMethod Supports a Single Value JSON null" {
+            $url = '{0}{1}' -f $baseUrl, 'null'
+            Invoke-RestMethod -Uri $url | Should Be $null
+        }
+        It "Invoke-RestMethod Supports a Single Value JSON null and ignores whitespace" {
```

Comment:
Perhaps, add test(s) for the error case where the JSON cannot be parsed?

Reply:
@TravisEz13 if by "error" you mean empty strings or whitspace-only? I can add the tests, but they will just be ensuring a a `String.Empty` returns a `String.Empty` and that a whitespace string returns a whitespace string. When that error condition hits, the cmdlet just goes to the next serialization type (xml) and then when that fails it just returns the string. 

It also looks like I would need  to either add a WebListener controller to another switch option to HttpListener. Currently none of the available tests can return whitespace only responses.  All of them truncate to 0 bytes when the body payload is all whitespace.

Reply:
It would be good to have the test to ensure good coverage, but I think this is adequate.

Reply:
I've created #5414 to add coverage for that. I have a backlog of test and cleanup activities to do, I'll start on those after RC is underway.